# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/moz.build
# Commit: 7cdf3cef2773
# Full Hash: 7cdf3cef27736a14fd8f9db58522da37e103d984
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-02 21:26:33
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 3: Add mechanism to notify EarlyHintPreloader that HttpChannelParent is ready to complete the redirect r=necko-reviewers,kershaw,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161178
# ==============================================================================

diff -r 5090eae24a5c -r 7cdf3cef2773 netwerk/protocol/http/moz.build
--- a/netwerk/protocol/http/moz.build	Fri Dec 02 09:45:24 2022 +0000
+++ b/netwerk/protocol/http/moz.build	Fri Dec 02 09:45:24 2022 +0000
@@ -51,6 +51,7 @@
     "ClassifierDummyChannelParent.h",
     "ClassOfService.h",
     "EarlyHintPreloader.h",
+    "EarlyHintRegistrar.h",
     "EarlyHintsService.h",
     "HttpAuthUtils.h",
     "HttpBackgroundChannelChild.h",
@@ -103,6 +104,7 @@
     "ConnectionHandle.cpp",
     "DnsAndConnectSocket.cpp",
     "EarlyHintPreloader.cpp",
+    "EarlyHintRegistrar.cpp",
     "EarlyHintsService.cpp",
     "Http2Compression.cpp",
     "Http2Push.cpp",